packages/atomic/src/components/facets-v1/atomic-rating-facet/atomic-rating-facet.tsx

```diff
@@ -0,0 +1,237 @@
+ *
+ * @part values - The facet values container.
+ * @part value-label - The facet value label, common for all displays.
+ * @part value-count - The facet value count, common for all displays.
+ *
+ * @part value-checkbox - The facet value checkbox, available when display is 'checkbox'.
+ * @part value-checkbox-label - The facet value checkbox clickable label, available when display is 'checkbox'.
+ * @part value-link - The facet value when display is 'link'.
+ * @part value-box - The facet value when display is 'box'.
+ *
+ * @part show-more - The show more results button.
+ * @part show-less - The show less results button.
+ * @part show-more-less-icon - The icons of the show more & show less buttons.
+ */
+@Component({
+  tag: 'atomic-rating-facet',
+  styleUrl: 'atomic-rating-facet.pcss',
+  shadow: true,
+})
+export class AtomicRatingFacet
+  implements
+    InitializableComponent,
+    BaseFacet<NumericFacet, NumericFacetState> {
+  @InitializeBindings() public bindings!: Bindings;
+  public facet!: NumericFacet;
+  public filter?: NumericFilter;
+  public searchStatus!: SearchStatus;
+
+  @BindStateToController('facet')
+  @State()
+  public facetState!: NumericFacetState;
+  @BindStateToController('filter')
+  @State()
+  public filterState?: NumericFilterState;
+  @BindStateToController('searchStatus')
+  @State()
+  public searchStatusState!: SearchStatusState;
+  @State() public error!: Error;
+  @State() public isCollapsed = false;
+
+  /**
+   * Specifies a unique identifier for the facet.
+   */
+  @Prop({mutable: true, reflect: true}) public facetId?: string;
+  /**
+   * The non-localized label for the facet.
+   */
+  @Prop() public label = 'rating';
+  /**
+   * The field whose values you want to display in the facet.
+   */
+  @Prop() public field!: string;
+  /**
+   * The number of values to request for this facet, when there are no manual ranges.
+   */
+  @Prop() public numberOfValues = 5;
+  /**
+   * The sort criterion to apply to the returned facet values.
+   * Possible values are 'ascending' and 'descending'.
+   */
+  @Prop() public sortCriteria: RangeFacetSortCriterion = 'ascending';
+  /**
+   * Whether to display the facet values as checkboxes (multiple selection) or links (single selection).
+   * Possible values are 'checkbox' and 'link'.
+   */
+  @Prop() public displayValuesAs: 'checkbox' | 'link' = 'checkbox';
+
+  public initialize() {
+    this.searchStatus = buildSearchStatus(this.bindings.engine);
+    this.initializeFacet();
+  }
+
+  private initializeFacet() {
+    const options: NumericFacetOptions = {
+      facetId: this.facetId,
+      field: this.field,
+      numberOfValues: this.numberOfValues,
+      sortCriteria: this.sortCriteria,
+      generateAutomaticRanges: false,
+    };
+    this.facet = buildNumericFacet(this.bindings.engine, {options});
+    this.facetId = this.facet.state.facetId;
+    this.bindings.store.state.facets[this.facetId] = {
```

Comment:
Once you pull master you'll have to implement the Atomic Store and format similar to the following https://github.com/coveo/ui-kit/pull/952